### ISSUE TITLE
Text wrapping does not work in DataGridColumnHeader

### ISSUE THREAD

**Issue by mgnslndh** [MaterialDesignInXAML/MaterialDesignInXamlToolkit#909]
Using `TextWrapping` on a `TextBlock` inside a `DataGridColumnHeader` does not work because the [control template uses a horizontal `StackPanel` ](https://github.com/ButchersBoy/MaterialDesignInXamlToolkit/blob/2740f14a814896d42032ae0013b765a8a0ec04c3/MaterialDesignThemes.Wpf/Themes/MaterialDesignTheme.DataGrid.xaml#L215)

## Example 

This header style will not wrap the content of the header.

```
<DataGridTextColumn.HeaderStyle>
    <Style TargetType="{x:Type DataGridColumnHeader}" BasedOn="{StaticResource MaterialDesignDataGridColumnHeader}">
        <Setter Property="HorizontalAlignment" Value="Right" />
        <Setter Property="ContentTemplate">
            <Setter.Value>
                <DataTemplate>
                    <TextBlock TextWrapping="Wrap" Text="{Binding}" TextAlignment="Right"/>
                </DataTemplate>
            </Setter.Value>
        </Setter>
    </Style>
</DataGridTextColumn.HeaderStyle>
```

**Comment by mgnslndh** [comment]
Submitted PR #910 to fix this

**Comment by Keboo** [comment]
Closed by #910
